bin/pycbc_make_sky_grid

```diff
@@ -85,9 +85,8 @@ def make_multi_det_grid(args):
     if (args.ra and args.dec and args.sky_error) is not None:
         # --ra/--dec/--sky-error arguments
         assert args.coverage is None
-        sky_dist = pycbc.distributions.UniformDiskSky(
-            mean_ra=args.ra, mean_dec=args.dec, radius=args.sky_error
-        )
+        args.input_dist = f"pycbc.distributions.UniformDiskSky(mean_ra={args.ra}, mean_dec={args.dec}, radius={args.sky_error})"
```

Comment:
The `pycbc.distributions.` prefix should not be included here as it is already added by `dist_from_str()`. Also, quotes are needed around the arguments, since they will be strings when the units are included (e.g. "1.234rad").
```suggestion
        args.input_dist = f'UniformDiskSky(mean_ra="{args.ra}", mean_dec="{args.dec}", radius="{args.sky_error}")'
```